tests/eosio.system_tester.hpp

```diff
@@ -1024,9 +1024,9 @@ class eosio_system_tester : public TESTER {
       }
       produce_blocks( 250 );
 
-      auto producer_keys = control->head_block_state()->active_schedule.producers;
-      BOOST_REQUIRE_EQUAL( 21, producer_keys.size() );
-      BOOST_REQUIRE_EQUAL( name("defproducera"), producer_keys[0].producer_name );
+      auto producer_keys = control->active_producers();
```

Comment:
strange that the variable is named `producer_keys` when it is a `producer_authority_schedule`

Reply:
Renamed